src/Perimeterx.php

```diff
@@ -91,7 +91,7 @@ private function __construct(array $pxConfig = [])
                 'send_block_activities' => true,
                 'sdk_name' => 'PHP SDK v2.6.1',
                 'debug_mode' => false,
-                'perimeterx_server_host' => 'https://sapi-' . strtolower($pxConfig['app_id']) . '.perimeterx.net',
+                'perimeterx_server_host' => 'https://collector-' . strtolower($pxConfig['app_id']) . '.perimeterx.net',
```

Comment:
you are right, it shoudn't be here, its only for mobile
the change need to be on the template only, not for the configuration
please view the ruby enforcer